packages/web/src/utils/route.ts

```diff
@@ -500,7 +503,7 @@ export const getPathname = (location: Location) => {
 
 export const recordGoToSignup = (callback: () => void) => {
   if ((window as any).analytics) {
-    ;(window as any).analytics.track(
+    ; (window as any).analytics.track(
```

Comment:
sup with this?

Reply:
think this was my linter bugging out. ill fix 👍